Makefile.local

```diff
@@ -0,0 +1 @@
+OCAMLWARN=-warn-error +a-3-8-40-32-28
```

Comment:
Coq 8.11 now turns all warnings into errors. This line is necessary to turn the errors (which were formerly warnings) back to warnings. I took a whitelist approach just to be safe, so only the following warnings: 3, 8, 40, 32, and 28 are kept as warnings.